userspace/libsinsp/filter/parser.cpp

```diff
@@ -0,0 +1,572 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <cstring>
+#include <regex>
+#include "parser.h"
+#include "../utils.h"
+#include "../sinsp_exception.h"
+
+using namespace libsinsp::filter;
+
+static const vector<string> unary_ops =
```

Comment:
You must be inheriting a "using namespace std;" somewhere. I'd suggest putting it in this file so we can someday remove any "using namespace"s from header files.